Lib/fontTools/ttx.py

```diff
@@ -362,7 +362,8 @@ def parseOptions(args):
 			output = makeOutputFileName(input, options.outputDir, extension, options.overWrite)
 			# 'touch' output file to avoid race condition in choosing file names
 			if action != ttList:
-				open(output, 'a').close()
+				with open(output, 'a') as f:
+					pass
```

Comment:
the change in this line is mostly cosmetic, and I actually prefer the previous one as it's more compact